content/00.build/30.zksync-sso/20.features.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Features
+description: Learn about the features of ZKsync SSO.
+---
+
+## Passkeys Authentication
+
+Passkeys are a core feature supported by ZKsync SSO that enable secure, passwordless authentication for users interacting with applications on the ZK Chain.
+Biometric authentication methods like Face ID and fingerprint recognition are supported by ZKsync SSO, utilizing the `secp256r1` elliptic curve.
+
+## Sessions
+
+A session is a temporary, limited-permission access to the account through a configurable policy.
+Sessions are explicitly approved by account owners and are managed via session keys.
+A session key owner has the ability to send a *certain restricted set* of transactions from the account without requiring the account owner’s signature.
+
+## Spend Limits
+
+**Spend limits** define the maximum amount of a specific token that can be spent during a session.
+They are an integral part of the session's policy and provide granular control over:
+
+- Token Type: ERC-20
+- Limit Amount: The maximum amount that can be transacted.
+
+When an application requests a session, it specifies the desired spend limits, which the account owner can approve.
+This ensures that even if the session key is compromised, the potential loss is capped.
+
+## Account Recovery
+
+There are several methods available to users to recover their account:
+
+**Multiple Devices with Passkey Synchronization**
+
+Users can synchronize their passkeys across multiple devices using services like iCloud Keychain for Apple devices or similar services on other platforms.
+This means if one device is lost, the passkeys are still accessible on other synchronized devices.
+
+**Adding another passkey — *coming soon***
+
+Users can register a new device by adding a new passkey to their account.
+This typically involves verifying their identity on the new device, after which the new passkey is securely associated with their account.
+
+**Recovery via Externally Owned Account (EOA) — *coming soon***
+
+Users will soon be able to link an EOA to their ZKsync SSO account.
+If passkeys are lost, they can recover access by authenticating with their EOA, providing an additional layer of security and recovery option.
+
+**Recovery via OAuth 2.0 — *coming soon***
+
+OAuth Recovery ERC-7579 module enables users to set up an account recovery through OAuth tokens associated with their existing Web2 logins like
+Gmail, Facebook, or X (formerly Twitter). This allows users to regain access by logging in with their familiar social media or email accounts.
+
+## User Dashboard
```

Comment:
Should we mention this if it's not ready?

Reply:
Commented out for now - coming soon :)